usecases/backup/coordinator.go

```diff
@@ -61,8 +61,8 @@ type participantStatus struct {
 	Reason   string
 }
 
-// selector is used to select participant nodes
-type selector interface {
+// Selector is used to select participant nodes
+type Selector interface {
```

Comment:
**Question:** why is this interface now publically exported? I can't seem to find any instances of `backup.Selector` in any of the diff 🤔

Reply:
that's exported to be able to test atm, this interface actually shall be removed not needed, but no time to do this refactor now